pcsx2/DEV9/AdapterUtils.h

```diff
@@ -31,7 +31,7 @@
 namespace AdapterUtils
 {
 #ifdef _WIN32
-	bool GetWin32Adapter(const std::string& name, PIP_ADAPTER_ADDRESSES adapter, std::unique_ptr<IP_ADAPTER_ADDRESSES[]>* buffer);
+	bool GetWin32Adapter(const char* name, PIP_ADAPTER_ADDRESSES adapter, std::unique_ptr<IP_ADAPTER_ADDRESSES[]>* buffer);
```

Comment:
Any reason from switching from `std::string&` to `char*`? Just curious

Reply:
So that we don't have to create new `std::string`s from `char*`s just to call the method

We can't use a `std::string_view` because these get passed to c functions which expect c (null-terminated) strings.

Reply:
Ah, that wasn't an issue before as anywhere this was called before either already had the name as a `std::string` or needed to create a `std::string` for another purpose anyway.

I see in this PR that you now call this function in `GetAdapterMAC()`, which admittedly would be easier to have take a `char*` vs a `std::string`